frontend/device/remarkable/device.lua

```diff
@@ -6,6 +6,7 @@ local function no() return false end
 
 local Remarkable = Generic:new{
     isRemarkable = yes,
+    model = "reMarkable",
```

Comment:
pretty much irrelevant for this commit, ignore if you want. 

The logic is to override device model even on generic types because the inherited model is nil, from https://github.com/koreader/koreader/blob/master/frontend/device/generic/device.lua#L21